gherkin/flagd.feature

```diff
@@ -0,0 +1,81 @@
+Feature: flagd providers
+
+  # This test suite contains scenarios to test flagd providers (both RPC and in-process).
+  # It's associated with the flags configured in flags/custom-ops.json and flags/zero-flags.json
+  # It should be used in conjunection with the suites supplied by the OpenFeature specification
+
+  Background:
+    Given a flagd provider is set
+
+  # events
+  Scenario: Provider ready event
+    When a PROVIDER_READY handler is added
+    Then the PROVIDER_READY handler must run
+
+  Scenario: Flag change event
+    When a PROVIDER_CONFIGURATION_CHANGED handler is added
```

Comment:
The test-harness containers (flagd and sync) both change this flag every 1s, so no action is required here except to attach a handler and wait 1s